negotiation/proposed.emu

```diff
@@ -359,22 +359,20 @@
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getstringorbooleanoption" aoid="GetStringOrBooleanOption">
-      <h1>GetStringOrBooleanOption ( _options_, _property_, _values_, _trueValue_, _falsyValue_, _fallback_ )</h1>
+    <emu-clause id="sec-getstringorbooleannumberformatoption" aoid="GetStringOrBooleanNumberFormatOption">
+      <h1>GetStringOrBooleanNumberFormatOption ( _options_, _property_, _values_, _fallback_ )</h1>
```

Comment:
I think renaming would better emphasis that boolean cases are addressed first, and that the third argument applies only after they fail to handle the input.
```suggestion
    <emu-clause id="sec-getbooleanorstringnumberformatoption" aoid="GetBooleanOrStringNumberFormatOption">
      <h1>GetBooleanOrStringNumberFormatOption ( _options_, _property_, _stringValues_, _fallback_ )</h1>
```
You should also consider adopting [Structured Headers](https://tc39.es/ecmarkup/#emu-clause-structured-headers), which would eliminate some redundancy.